docs/projects/ad4630_fmc/index.rst

```diff
@@ -61,13 +68,16 @@ Supported boards
 -  :adi:`EVAL-AD4030-24FMCZ <EVAL-AD4030-24FMCZ>`
 -  :adi:`EVAL-AD4630-16FMCZ <EVAL-AD4630-16FMCZ>`
 -  :adi:`EVAL-AD4630-24FMCZ <EVAL-AD4630-24FMCZ>`
+-  EVAL-ADAQ4224-FMCZ <EVAL-ADAQ4224>
```

Comment:
- [ ] Lines 71, 72 and 80 are not made as links, and they should be.
- [ ] Also, please add link to this documentation in docs/projects/index.rst